### DISCUSSION TITLE
Best approach to style with tailwind?

### DISCUSSION THREAD

**Discussion by wrabit** [orchidjs/tom-select#196] [Q&A]
Would like to use tailwind utility classes to style the dropdown like other fields..

![image](https://user-images.githubusercontent.com/5918271/131811754-db852c17-5c52-470d-8a91-bcbd57158317.png)

**Comment by jrief** [comment]
Maybe [this wrapper](https://github.com/orchidjs/tom-select/discussions/296) might be interesting for you.
